app/src/main/java/org/jellyfin/androidtv/ui/presentation/CardPresenter.java

```diff
@@ -191,7 +192,18 @@ public void setItem(BaseRowItem m, String imageType, int lHeight, int pHeight, i
                     UserItemDataDto userData = itemDto.getUserData();
                     if (showWatched && userData != null) {
                         if (userData.getPlayed()) {
-                            mCardView.setUnwatchedCount(0);
+                            WatchedIndicatorBehavior showIndicator = get(UserPreferences.class).get(UserPreferences.Companion.getWatchedIndicatorBehavior());
+                            if (showIndicator == WatchedIndicatorBehavior.EPISODES_ONLY) {
+                                if (itemDto.getBaseItemType() == BaseItemType.Episode) {
+                                    mCardView.setUnwatchedCount(0);
+                                }
+                                else {
+                                    mCardView.setUnwatchedCount(-1);
+                                }
+                            }
+                            else {
+                                mCardView.setUnwatchedCount(0);
+                            }
```

Comment:
```suggestion
                            if (showIndicator == WatchedIndicatorBehavior.EPISODES_ONLY && itemDto.getBaseItemType() != BaseItemType.Episode) {
                                    mCardView.setUnwatchedCount(-1);
                             } else {
                                    mCardView.setUnwatchedCount(0);
                             }
```

Could be simplified to something like this